### ISSUE TITLE
Apple silicon machines require Rosetta to be installed to run CEA

### ISSUE THREAD

**Issue by reyery** [architecture-building-systems/CityEnergyAnalyst#3120]
Currently the python environment for CEA on Apple silicon machines is based on x86 due to some dependencies not found for osx-arm.

Python modules that prevent CEA from running natively on Apple silicon machines (according to conda):
```
  - flask-restplus
  - wntr 0.2.2.1**
  - numpy 1.19.1**
  - pythonocc-core
```

Also found out that daysim binaries are running under x86 even though they are compiled as Universal binaries. Probably due to python running bash as x86.

**Comment by reyery** [comment]
Update: `pythonocc-core` is now available for `osx-arm64` in conda-forge

https://github.com/tpaviot/pythonocc-core/issues/1071

**Comment by jimenofonseca** [comment]
Hey rey...and what happened to Wntr? I see that the thermal network does not run on Master for M1-2 CHIPS.

**Comment by reyery** [comment]
@jimenofonseca You might need to update the conda/python environment since there were changes. You could try installing it for Mac again from the docs using the new instructions [here](https://city-energy-analyst.readthedocs.io/en/latest/installation/installation-on-macos.html).

**Comment by jimenofonseca** [comment]
Yes. I did a fresh install in a new mac. There i found the problem. How is it in your end? Does it work the network simulation on Master in a mac?JimenoOn 10 Mar 2024, at 01:32, Reynold Mok ***@***.***> wrote:﻿
@jimenofonseca You might need to update the conda/python environment since there were changes. You could try installing it for Mac again from the docs using the new instructions here.

**Comment by reyery** [comment]
@jimenofonseca I have tested on two different apple silicon macs and I was able to run every script. What was the actual error that you are getting? I can try to reproduce it.

**Comment by jimenofonseca** [comment]
Hoi rey i got the problem, that the class _elevated in wntr does not exist.Do you see this also?JimenoOn 18 Mar 2024, at 08:12, Reynold Mok ***@***.***> wrote:﻿
Closed #3120 as completed.

**Comment by reyery** [comment]
@jimenofonseca that issue was resolved in https://github.com/architecture-building-systems/CityEnergyAnalyst/pull/3265/commits/c60e7dbaa63d8d7390a3fe4804fd8dd0149e2ba8

Could you check what version of `wntr` is in your environment?
